openssl/src/ec.rs

```diff
@@ -527,6 +527,30 @@ impl EcPointRef {
             .map(|_| ())
         }
     }
+
+    /// Checks if point is infinity
```

Comment:
The docs should mention what these correspond to like on the other methods in the module.

Reply:
added